docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1353,6 +1353,45 @@ Then value `$value` is selected in slider located by `$locator`
 Then value `50` is selected in slider located by `id(test_slider)`
 ----
 
+=== Emulation steps
+
+==== Override device metrics
+
+Overrides the values of device screen dimensions.
```

Comment:
which browsers are supported?
what is an effect of this step execution?

the same for the step clearing metrics

Reply:
It turns out these steps are supported only on chrome browser, for other browsers:
* non-chromium browser like safari: class case exception during augumentation
* chromium browsers: Exception in thread "main" org.openqa.selenium.UnsupportedCommandException: unknown command: unknown command: session/6f209346788418b9ce82c8619162e1df/goog/cdp/execute